charmcraft/templates/init-flask-framework/charmcraft.yaml.j2

```diff
@@ -0,0 +1,31 @@
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
```

Comment:
While the extension does fill in the `type` field, I'd prefer if we keep it in the template too, since a user might see from the documentation that it's required and be confused by it. This confusion will become more obvious when we make a jsonschema for `charmcraft.yaml`, since that will mark `type` as required (and thus IDEs will start complaining about its nonexistence).

Reply:
Gotcha, added, thanks!